Tests/SentryTests/SentryViewPhotographerTests.swift

```diff
@@ -144,6 +144,22 @@ class SentryViewPhotographerTests: XCTestCase {
         assertColor(pixel2, .green)
     }
     
+    func testRedactLabelWithParentTransformed() throws {
+        let label = UILabel(frame: CGRect(x: 0, y: 0, width: 50, height: 25))
+        label.text = "Test"
+        let parentView = UIView(frame: CGRect(x: 0, y: 17, width: 50, height: 25))
+        parentView.backgroundColor = .green
+        parentView.transform = CGAffineTransform(rotationAngle: 90 * .pi / 180.0)
+        parentView.addSubview(label)
+        
+        let image = try XCTUnwrap(prepare(views: [parentView] ))
+        let pixel1 = color(at: CGPoint(x: 10, y: 10), in: image)
+        assertColor(pixel1, .white)
+        
+        let pixel2 = color(at: CGPoint(x: 22, y: 10), in: image)
+        assertColor(pixel2, .black)
```

Comment:
> As long as the general region is masked

The test only checks two pixels. How can you be sure the whole region is masked correctly in this scenario?

> and we have other tests to check the rectangle that is being masked

Also, when using `transform`?

Reply:
I guess we could add another test with transformation in `UIRedactBuilderTests` that is responsible for the calculation of the region.

Reply:
I decided to test in the ViewPhotographer because transformation, especially rotation, needs to work with long float numbers, and dealing with floating imprecision can cause flakiness.

So, it will be easier to test with the final image.

Reply:
For testing readability, you could use the exact expected values and create an assert helper method that allows some deviation.

Reply:
Maybe in another PR. My feedback is that it's hard to read these tests. You have to mentally map the coordinates to check if it's working correctly.